glide.yaml

```diff
@@ -1,32 +1,31 @@
 package: github.com/uber-go/dosa
 import:
+- package: github.com/elodina/go-avro
+- package: github.com/golang/mock
+  subpackages:
+  - gomock
+- package: github.com/jessevdk/go-flags
+  version: ^1.2.0
 - package: github.com/pkg/errors
   version: ^0.8.0
 - package: github.com/satori/go.uuid
   version: ^1.1.0
-- package: github.com/elodina/go-avro
 - package: github.com/uber/dosa-idl
   subpackages:
-  - .gen
+  - .gen/dosa
+  - .gen/dosa/dosaclient
+- package: go.uber.org/yarpc
+  version: ^1.7.1
+  subpackages:
+  - api/transport
+  - transport/http
+  - transport/tchannel
+testImport:
+- package: github.com/yookoala/realpath
 - package: github.com/stretchr/testify
   version: ^1.1.4
   subpackages:
   - assert
-  - mock
-- package: github.com/golang/mock
```

Comment:
I believe it's actually needed up here since we supply our own mocks. when I ran through glide's recommendations regarding our `glide.yaml` file, it said this belongs up here.